test/e2e/usb-protected.e2e.js

```diff
@@ -0,0 +1,440 @@
+const capitalize = require('lodash/capitalize');
```

Comment:
Comment for this file as a whole. I intentionally included a few redundant checks in the `after` of the previous describe block as well as `before` of the current describe block for increases robustness